sdcm/nemesis.py

```diff
@@ -1005,8 +1006,19 @@ def _destroy_data_and_restart_scylla(self):
                 if result.stderr:
                     raise FilesNotCorrupted('Files were not corrupted. CorruptThenRepair nemesis can\'t be run. '
                                             'Error: {}'.format(result))
+                deleted_files_counter += 1
                 self.log.debug('Files {} were destroyed'.format(file_for_destroy))
-
+        except NoFilesFoundToDestroy:
+            if deleted_files_counter > 0:
+                InfoEvent(
+                    message=(f"Found only '{deleted_files_counter}' files "
+                             "out of 5 to be deleted/corrupted"),
+                    severity=Severity.WARNING,
+                ).publish()
+            else:
+                # Skip this nemesis because the precondition is not met
+                raise UnsupportedNemesis(  # pylint: disable=raise-missing-from
```

Comment:
This is problematic, we've seen this week it can fail cause of issues we don't want skip, issues that we want to fix.

It was failing on timeouts ? or on it was run before we had enough data ?

Reply:
It failed to find a file for deletion on the line with `_choose_file_for_destroy()` method call.

Reply:
Cause we don't have enough data written before this nemesis start ? If so we have a feature in SCT longevity to wait for having some size written before starting nemesis.

And if we are running nemesis during prepare, let's stop doing that for now.

Reply:
It happens not during prepare. It happens during the common load. In current case it is `write` load.

Reply:
> It happens not during prepare. It happens during the common load. In current case it is `write` load.

I'm guessing
 
```yaml
space_node_threshold: 64424
```

it's enough for having 5 sstables